lib/standard/stream.nit

```diff
@@ -220,6 +220,64 @@ abstract class Reader
 	# Is there something to read.
 	# This function returns 'false' if there is something to read.
 	fun eof: Bool is abstract
+
+	# Read the next sequence of non whitespace characters.
+	#
+	# Leading whitespace characters are skipped.
+	# The first whitespace character that follows the result is consumed.
+	#
+	# An empty string is returned if the end of the file or an error is encounter.
```

Comment:
Both ways of doing the error/ending check are valid, I like the actual way of doing things since it prevents from having to do null type checks before calling a method on the receiver, it's a bit less error-prone.
But if there's a general consensus to change to the `nullable` way, I won't be fighting against it.